js/web/lib/onnxjs/backends/webgl/ops/matmul.ts

```diff
@@ -18,23 +20,33 @@ export class WebGLMatMul extends MatMul implements WebGLOperator {
   }
 
   run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
-    if (inferenceHandler.session.pack) {
+    if (this.usePackedTexture === undefined) {
+      const isBroadcast = !ShapeUtil.areEqual(inputs[0].dims, inputs[1].dims);
+      this.usePackedTexture = !isBroadcast && inferenceHandler.session.pack;
+    }
+
+    if (this.usePackedTexture === true) {
```

Comment:
`if (this.usePackedTexture === true)` can be simplified to `if (this.usePackedTexture) `

Reply:
updated